src/cata_tiles.cpp

```diff
@@ -270,6 +270,11 @@ void cata_tiles::load_tilejson_from_file(std::ifstream &f, const std::string &im
         JsonObject curr_info = info.next_object();
         tile_height = curr_info.get_int("height");
         tile_width = curr_info.get_int("width");
+        if (curr_info.has_bool("iso")) {
```

Comment:
`x = json.get_bool( "...", false )` The second parameter to the various `get_*` functions is the default value.